src/osx/Installer.Mac/uninstall.sh

```diff
@@ -12,7 +12,7 @@ fi
 
 # Unconfigure (as the current user)
 echo "Unconfiguring credential helper..."
-sudo -u `/usr/bin/logname` "$GCMBIN" unconfigure
+sudo -u `/usr/bin/logname` HOMEBREW_PREFIX=$HOMEBREW_PREFIX "$GCMBIN" unconfigure
```

Comment:
Aha! The reason `HOMEBREW_PREFIX` wasn't coming through to the GCM process was because we're going via `sudo`. Perhaps we should instead use the following option to preserve _all_ environment variables?

```man
-E, --preserve-env
                   Indicates to the security policy that the user wishes to preserve their existing
                   environment variables.  The security policy may return an error if the user does not have
                   permission to preserve the environment.

```